docs/jwst/outlier_detection/reference_files.rst

```diff
@@ -0,0 +1,4 @@
+Reference Files
+===============
+
+The ``outlier_detection`` step uses the PARS-OUTLIERDETECTIONSTEP parameter reference file.
```

Comment:
I think for other steps, if it does not use any non-parameter reference files, we usually say it does not use any reference files.  There are none that the user needs to consider or potentially override.